test/OpenTelemetry.Exporter.Geneva.Benchmark/Exporter/TraceExporterBenchmarks.cs

```diff
@@ -74,28 +78,12 @@ public TraceExporterBenchmarks()
                     ["cloud.roleVer"] = "9.0.15289.2",
                 },
             });
-
-            this.tracerProvider = Sdk.CreateTracerProviderBuilder()
-                .SetSampler(new AlwaysOnSampler())
-                .AddSource(this.activitySource.Name)
-                .AddGenevaTraceExporter(options =>
-                {
-                    options.ConnectionString = "EtwSession=OpenTelemetry";
-                    options.PrepopulatedFields = new Dictionary<string, object>
-                    {
-                        ["cloud.role"] = "BusyWorker",
-                        ["cloud.roleInstance"] = "CY1SCH030021417",
-                        ["cloud.roleVer"] = "9.0.15289.2",
-                    };
-                })
-                .Build();
         }
 
         [Benchmark]
         public void ExportActivity()
         {
-            // this activity will be created and feed into the actual Geneva exporter
-            using var activity = this.activitySource.StartActivity("Benchmark");
+            this.exporter.Export(this.batch);
```

Comment:
this looks same as the SerializeActivity benchmark? Do we really need this, since we already have a dedicated one just to measure serialization cost?

Reply:
I think this does provide some additional information. It shows we are not doing any allocation in the `Export` method other than what's required for serialization. I actually wanted to add this to compare it with ETW/TLD Exporter.

I could add back the original benchmark which includes instrumentation +export along with this?